spec/tic_tac_toe/board_spec.clj

```diff
@@ -53,3 +51,13 @@
     (should= false (full? empty-board-3x3))
     (should= false (full? test-board))
     (should= true (full? full-board-3x3))))
+
+(describe "valid position"
+  (it "should return true if position is a valid board position"
+    (should= true (valid-position? test-board "1"))
+    (should= true (valid-position? test-board "8")))
```

Comment:
I would try to have two assertions here. If you can't think of another `it` statement for an assertion it might not be necessary.